master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -1,142 +1,102 @@
 ---
-title: Going Beyond `NetworkPolicy` with Calico
+title: Going Beyond NetworkPolicy with Calico
 ---
 
-The Kubernetes NetworkPolicy API allows users to express ingress policy to Kubernetes pods
-based on labels and ports.  Calico implements this API, but also supports a number of
-policy features which are not currently expressble through the NetworkPolicy API such as CIDR
-and egress policy.
+The Kubernetes `NetworkPolicy` API allows users to express ingress and egress policies (starting with Kubernetes 1.8.0) to Kubernetes pods
+based on labels and ports.
 
-This guide walks through using the Calico APIs directly in conjunction with Kubernetes NetworkPolicy
-in order to define more complex network policies.
+This guide walks through using only the Kubernetes `NetworkPolicy` in order to define more complex network policies.
```

Comment:
This sentence is a bit awkward now - I'd suggest removing the "only the" and "in order" to make it read a bit better.

So, "This guide walks through using Kubernetes NetworkPolicy to define"